File: Study-Code/Some_Function_C++/Search/Linear_search.cpp
#include <iostream>
using namespace std;

int linear_search(int arr[],int n,int x) 
{
  for(int i = 0; i < n; i++)
    {
      if(arr[i] == target)
        return i;
    }
  return -1;
}

int main()
{
  int n, target; 
  cin >> n >> target;
  int arr[n];

  for(int i = 0; i < n; i++) 
  {
    cin >> arr[i];
  }

  int res = linear_search(arr,n,target);
  cout << res;
  
  return 0;
}

File: Study-Code/Some_Function_C#/Search_Algorithm/Linear_Search.cs
using System;
using System.ComponentModel;

namespace Practice_C_Sharp
{
    internal class Program
    {
        static int Linear_Search(int[] arr, int size, int target)
        {
            for(int i = 0; i < size; i++)
            {
                if (arr[i] == target)
                    return i;
            }
            return -1;
        }
        public static void Main(string[] args)
        {
            int size = Convert.ToInt32(Console.ReadLine());
            int target = Convert.ToInt32(Console.ReadLine());
            int[] array = new int[size];

            for(int  i = 0; i < size; i++)
            {
                array[i] = Convert.ToInt32(Console.ReadLine());
            }

            int result = Linear_Search(array, size, target);
            Console.Write($"Position of target is: {result}");

        }
    }
}
